build/pkgs/pari_jupyter/distros/conda.txt

```diff
@@ -1 +0,0 @@
-pari_jupyter
```

Comment:
This should be fixed now. `pari_jupyter` sdist was distributing a cythonized file that didn't work for python3.11.

Reply:
Thanks! I've put it back in.